libnetdata/socket/socket.c

```diff
@@ -301,14 +301,37 @@ void listen_sockets_close(LISTEN_SOCKETS *sockets) {
     sockets->failed = 0;
 }
 
+WEB_CLIENT_ACL socket_ssl_acl(char *ssl){
```

Comment:
valgrind: Uninitialized value & jump condition in line 988. I sent you the valgrind.xml

Reply:
I will talk more about it with you.